src/server/services/hli/hli.ts

```diff
@@ -39,7 +45,7 @@ function getDisplayStatus(
     case !aanvraag.isActueel && aanvraag.resultaat !== 'toegewezen':
       return 'Afgewezen';
   }
-  return statusLineItems[statusLineItems.length - 1].status ?? 'NNB';
+  return statusLineItems[statusLineItems.length - 1]?.status ?? 'Onbekend';
```

Comment:
Is dit een scenario wat eigenlijk nooit hoort voor te komen? 
Zo ja is het mogelijk om ons team hiervan op de hoogte te stellen op 1 of andere manier? Wellicht in AppIsights oid een automatische trigger bouwen met email notificatie als ergens `Onbekend` voor komt.

Reply:
Op zich kan dat wel maar heeft wmb geen hoge prio. Ik maak er een story van.